vividus-bdd-engine/src/main/java/org/vividus/bdd/converter/SubStepsConverter.java

```diff
@@ -53,8 +56,12 @@ public SubStepsConverter(ExtendedConfiguration configuration, IBddRunContext bdd
                         .collect(collectingAndThen(Collectors.toList(), DelegatingStoryReporter::new));
             }
             ExamplesTable subStepsTable = configuration.examplesTableFactory().createExamplesTable(subSteps);
-            List<Step> steps = stepExamplesTableParser.parse(subStepsTable,
-                    bddRunContext.getRunningStory().getRunningScenario().getExample());
+
+            Map<String, String> parameters = Optional.ofNullable(bddRunContext.getRunningStory().getRunningScenario())
```

Comment:
Optional.ofNullable(bddRunContext.getRunnignStory()).map(..) ?

Reply:
bddRunContext.getRunnignStory() - can it be null?

Reply:
You never know :)